client/js/components/main/assessment.jsx

```diff
@@ -94,24 +94,24 @@ export class Assessment extends React.Component{
     if(props.questionCount === undefined || index >= props.questionCount || index < 0){
       return <div></div>;
     }
-
-    return (
-      <Item
-          key              = {index /* react uses this to distinguish children */}
-          assessment       = {props.assessment}
-          settings         = {props.settings}
-          question         = {props.allQuestions[index]}
-          currentItemIndex = {index}
-          questionCount    = {props.questionCount}
-          messageIndex     = {props.progress.answerMessageIndex[index]}
-          allQuestions     = {props.allQuestions}
-          studentAnswers   = {{/*this.props.studentAnswers*/}}
-          outcomes         = {props.outcomes}
-          goToNextQuestion = {() => {props.nextQuestion();}}
-          goToPrevQuestion = {() => {props.previousQuestion();}}
-          submitAssessment = {() => {this.submitAssessment();}}
-      />
-    );
+    debugger;
+    return <Item
```

Comment:
Put the parentheses back.  Joseph and Ben as well as I settled on it for several reasons.  Mine is that my editor then does nice auto-formatting.  I don't know their reasons.